### ISSUE TITLE
Maybe I am a retard, but autojump doesn't seem to work on Natty Narwhal

### ISSUE THREAD

**Issue by ghost** [wting/autojump#65]
Hello, 

I installed autojump in Ubuntu 11.04 from synaptic and I can't get it to work. I added source /etc/profile in my bashrc,
closed and opened my terminal again, and whenever I try to use jumpstat it says command not found. 
I tried to train jumpstat by navigating a bit in my filesystem, but when I try j derp to go to a directory, it also says j: command not found.

Am I missing something?

**Comment by baryaqod** [comment]
Thanks. Same problem solved in Ubuntu Debian 8.

1. adding . /usr/share/autojump/autojump.sh ( remember to give whitespace after .   :) ) , as told in /usr/share/doc/autojump/README.Debian. ( if you dont find in /usr/share/ in your system. you can download the .deb file and read the README.Debian. 

2. source ~/.bashrc or  ~/.bash_profile

3. Solved :).

thanks everyone :)

//before, i thought some thing wrong in my instalation, but for Debian we have to do it manually.

**Comment by wting** [comment]
Yeah, I wish we could do it automatically for users but that runs afoul of Debian's guidelines. :crying_cat_face:
